portfolio/src/main/webapp/script.js

```diff
@@ -61,9 +61,8 @@ function setComments() {
   fetch(
       '/data' +
       '?num-comments=' + num_comments + '&order=' + order)
-      .then((response) => response.json())
+      .then((response) => {console.log(response); return response.json()})
```

Comment:
Nit: Please use multi-line block style whenever you have multiple statements:

```
...
.then(response => {
    console.log(response);
    return response.json();
})
...
```

Reply:
Consider using [`async/await`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) to remove nested `then`s. This makes your code more readable and should be easy to do  since for the remainder of this function, you're operating linearly on the response results.

You can do this by translating any `then`able object into an `await` call. Since your function uses `await` statements, the overall function needs to be marked as `async`.

For example:

```
async function setComments() {
  ...
  let obj = await fetch('/data', ...); // consider renaming 'obj' to 'comments'
  commentContainer = document.getElementById('comments-container');
  commentContainer.innerHTML = '';
  ...
}
```

No need to do this in this PR (or at all). I just wanted to point out that this other way of handling async results.